.github/workflows/cpath-pr.yml

```diff
@@ -126,9 +105,8 @@ jobs:
         with:
           lfs: true
 
-      - name: Set up smoke test environment
-        id: setup-slides-smoke-test-environment
-        uses: ./.github/actions/prepare_smoke_test_environment
+      - name: Prepare Conda environment
+        uses: ./.github/actions/prepare_cpath_environment
```

Comment:
replace the "prepare smoke test env" sections further down (from trainsfer_main) with  this segment here